assets/src/edit-story/components/tablist/tablistPanel.js

```diff
@@ -56,31 +57,38 @@ const TablistPanel = ({
 
   return (
     <PanelWrapper className={className} isExpanded={isExpanded}>
-      <TabButton
-        aria-controls={panelId}
-        aria-label={sprintf(
-          /* translators: %d: number of issues, %s: title */
-          _n('%1$d %2$s issue', '%1$d %2$s issues', badgeCount, 'web-stories'),
-          badgeCount,
-          title
-        )}
-        aria-selected={isExpanded}
-        onClick={onClick}
-        role="tab"
-        status={status}
-      >
-        <ButtonText>
-          <IconContainer>
-            <Icons.ChevronDownSmall />
-          </IconContainer>
-          <PanelText id={`${title}-${panelId}`} aria-hidden>
-            {title}
-          </PanelText>
-        </ButtonText>
-        <Badge>
-          <PanelText aria-hidden>{badgeCount}</PanelText>
-        </Badge>
-      </TabButton>
+      <TabButtonWrapper>
+        <TabButton
+          aria-controls={panelId}
+          aria-label={sprintf(
+            /* translators: %d: number of issues, %s: title */
```

Comment:
Pascal improved this in #8212 (https://github.com/google/web-stories-wp/pull/8212/files#diff-19a042f78fb750578fdbbfb643e3051db5b609aef6d8281ce220a1c610a7d75a) with 
` /* translators: 1: number of issues. 2: type of issue, for example "High Priority". */` - just a heads up to make sure that change sticks